.packit.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+# Build targets can be found at:
+# https://copr.fedorainfracloud.org/coprs/rhcontainerbot/packit-builds/
+
+specfile_path: podman.spec
```

Comment:
On other branches, this is `rpm/podman.spec` and there's the `update-spec-version` script.

On this branch, I don't actually see a `podman.spec`, just `podman.spec.rpkg`.

I still have no clue about packit, but is this a problem? If it is not a problem, and this is ready for actual review, lmk and I'll take another look. Otherwise I will assume this is still WIP.

Reply:
The `podman.spec.rpkg` was there already on that branch. I could switch to `rpm/podman.spec` but that might be too much work for a maint branch where IIUC, our main concern is if the PR builds as an rpm and we don't care about delivering that spec file anywhere downstream.

But if `rpm/podman.spec` gets us more consistency, I could probably do that. Just that we might need to consider the spec files for rhel 8 and 9 from rhel dist-git, which may have diverged (no idea).

Reply:
On second thoughts, it might be beneficial to use `rpm/podman.spec` and also use the same spec to build the official rhel packages to ensure full consistency. @jnovy wdyt ?

Reply:
@lsm5 I feel really stupid, and there's probably a super-obvious answer, but my question is:
1. The newly-added `.packit.yaml` refers to `podman.spec`
2. There is no file named `podman.spec` anywhere in this branch

Where is this `podman.spec` coming from?

Reply:
@lsm5 I'd keep only the current spec for the current latest release for podman. To maintain all conditionals for all mainline distros in one spec seems to be an overkill to me.